src/main/java/com/hivemq/bootstrap/netty/ChannelHandlerNames.java

```diff
@@ -108,26 +108,17 @@
 
     public static final String PLUGIN_INITIALIZER_HANDLER = "plugin_initializer_handler";
     public static final String CLIENT_LIFECYCLE_EVENT_HANDLER = "client_lifecycle_event_handler";
-    public static final String PUBLISH_OUTBOUND_INTERCEPTOR_HANDLER = "publish_outbound_interceptor_handler";
-    public static final String PUBACK_INTERCEPTOR_HANDLER = "puback_interceptor_handler";
-    public static final String PUBREC_INTERCEPTOR_HANDLER = "pubrec_interceptor_handler";
-    public static final String PUBREL_INTERCEPTOR_HANDLER = "pubrel_interceptor_handler";
-    public static final String PUBCOMP_INTERCEPTOR_HANDLER = "pubcomp_interceptor_handler";
     public static final String CONNECT_INBOUND_INTERCEPTOR_HANDLER = "connect_inbound_interceptor_handler";
-    public static final String CONNACK_OUTBOUND_INTERCEPTOR_HANDLER = "connack_outbound_interceptor_handler";
-    public static final String DISCONNECT_INTERCEPTOR_HANDLER = "disconnect_interceptor_handler";
-    public static final String SUBACK_OUTBOUND_INTERCEPTOR_HANDLER = "suback_outbound_interceptor_handler";
-    public static final String UNSUBSCRIBE_INBOUND_INTERCEPTOR_HANDLER = "inbound_unsubscribe_interceptor_handler";
-    public static final String UNSUBACK_OUTBOUND_INTERCEPTOR_HANDLER = "unsuback_outbound_interceptor_handler";
-    public static final String PING_INTERCEPTOR_HANDLER = "ping_interceptor_handler";
 
     /* *************
      *     Misc    *
      ***************/
 
     public static final String EXCEPTION_HANDLER = "exception_handler";
     public static final String CONNECTION_LIMITER = "connection_limiter";
-}
+    public static final String INTERCEPTORS = "interceptors";
```

Comment:
```suggestion
    public static final String INTERCEPTOR_HANDLER = "interceptor_handler";
```

Reply:
The shorter a handler name is the better (as handler can be inserted by name).
Also the "_handler" postfix is redundant as all names are for handlers.

Reply:
then change it everywhere and not in just one place